app/src/main/java/io/apicurio/registry/storage/impl/sql/AbstractSqlRegistryStorage.java

```diff
@@ -1082,13 +1082,17 @@ public ArtifactSearchResultsDto searchArtifacts(Set<SearchFilter> filters, Order
                         //    Note: convert search to lowercase when searching for properties (case-insensitivity support).
                         String propKey = property.getKey().toLowerCase();
                         String propValue = property.getValue().toLowerCase();
-                        where.append("EXISTS(SELECT p.globalId FROM properties p WHERE p.pkey = ? AND p.pvalue = ? AND p.globalId = v.globalId AND p.tenantId = v.tenantId)");
+                        where.append("EXISTS(SELECT p.globalId FROM properties p WHERE p.pkey = ? ");
                         binders.add((query, idx) -> {
                             query.bind(idx, propKey);
                         });
-                        binders.add((query, idx) -> {
-                            query.bind(idx, propValue);
-                        });
+                        if(propValue != ""){
```

Comment:
So it took me a little while to realize why this is working.  :)  At first I thought there was no way this could work because comparing two strings in this way is not meaningful unless they are both first interned.  But then I realized that the only time that `propValue` would be equal to the empty string is in the case where no value was provided.  Since Java interns all string literals, this check actually does work.  But I do think it's still a little weird.

Everything else is perfect, but I'm going to change this from empty string to `null`, which should be a bit more explicit. :)